test.sh

```diff
@@ -73,8 +73,12 @@ function setup_osbs() {
 
   # CentOS needs to have setuptools updates to use wildcards in requirements.txt and to make pytest-cov work
   if [[ $OS != "fedora" ]]; then
-    $RUN $PIP install -U setuptools
-
+    if [[ ${PYTHON_VERSION} == '2' ]]; then
```

Comment:
In this case you can just always upgrade pip, don't care about distro

Reply:
OK, wasn't sure if we had some reason for *not* doing it...